### PR TITLE
E UI 1003 mo unit test

### PR DESCRIPTION
**Pull request by waynepHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-1003

### Change description ###
improving the coverage for unit testing manage organisation app 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/fee-accounts/containers/overview-container/account-overview-container.component.ts [hunk 1]
```diff
@@ -1,24 +1,18 @@
 import { Component, Input, OnInit } from '@angular/core';
```
Comment (rayliangatos) [inline src/fee-accounts/containers/overview-container/account-overview-container.component.ts]: OnInit can be removed

File: src/fee-accounts/containers/overview/account-overview.component.spec.ts [hunk 2]
```diff
@@ -1,19 +1,139 @@
...
 
 describe('OrganisationAccountsComponent', () => {
     let component: OrganisationAccountsComponent;
-    const feeStore = jasmine.createSpyObj('feeStore', ['pipe']);
-    const organisationStore = jasmine.createSpyObj('feeStore', ['pipe']);
-    const actions = jasmine.createSpyObj('feeStore', ['pipe']);
-    const routerStore = jasmine.createSpyObj('feeStore', ['pipe']);
+    let mockStore: any;
 
+    let fixture: ComponentFixture<OrganisationAccountsComponent>;let superNgOnDestroy: jasmine.Spy
```
Comment (rayliangatos) [inline src/fee-accounts/containers/overview/account-overview.component.spec.ts:17]: can this let split to two lines and ; is missing

File: src/fee-accounts/containers/overview/account-overview.component.spec.ts [hunk 3]
```diff
@@ -1,19 +1,139 @@
...
+    beforeEach(async(() => {
+       
+        mockStore = jasmine.createSpyObj('Store', ['pipe', 'dispatch']);
+        let actions$ = of (
+          [
+            OrgActions.LOAD_ORGANISATION, OrgActions.LOAD_ORGANISATION_SUCCESS,
+            fromAccountStore.FeeAccountsEffects, fromAccountStore.LOAD_FEE_ACCOUNTS, fromAccountStore.LOAD_FEE_ACCOUNTS_FAIL,
+            fromAccountStore.LOAD_FEE_ACCOUNTS_SUCCESS, fromAccountStore.LOAD_FEE_ONE_OR_MORE_ACCOUNTS_FAIL, fromAccountStore.LOAD_FEE_RESET_STATE,
+
+          ])
```
Comment (rayliangatos) [inline src/fee-accounts/containers/overview/account-overview.component.spec.ts:68]: ; is missing

File: src/unassigned-cases/containers/unassigned-cases/unassigned-cases.component.spec.ts [hunk 4]
```diff
@@ -1,16 +1,126 @@
...
+      }],
+      loading: false,
+      error: undefined,
+      users: []
+    }
+  };
+
+  const matTabGroupElement = () => {
+    return fixture.debugElement.queryAll(By.css('mat-tab-group'));
+  }
```
Comment (rayliangatos) [inline src/unassigned-cases/containers/unassigned-cases/unassigned-cases.component.spec.ts:56]: mising ;

File: src/users/containers/invite-user/invite-user.component.ts [hunk 5]
```diff
@@ -184,6 +184,7 @@ export class InviteUserComponent implements OnInit, OnDestroy {
     }
   }
 
+  // tested
```
Comment (rayliangatos) [inline src/users/containers/invite-user/invite-user.component.ts:187]: removed it

File: src/unassigned-cases/converters/case-converter.ts [hunk 6]
```diff
@@ -4,6 +4,7 @@ import { SharedCase } from '@hmcts/rpx-xui-common-lib/lib/models/case-share.mode
 const BLANK_SPACE: string = ' ';
 const EMPTY_SPACE: string = '';
 const VERSUS_SPACE: string = ' Vs ';
+// tested
```
Comment (rayliangatos) [inline src/unassigned-cases/converters/case-converter.ts:7]: need to remove

File: src/unassigned-cases/containers/unassigned-cases/unassigned-cases.component.spec.ts [hunk 7]
```diff
@@ -1,16 +1,126 @@
...
+
+  const ccsListElement = () => {
+    return fixture.debugElement.queryAll(By.css('ccd-case-list'));
+  }
```
Comment (rayliangatos) [inline src/unassigned-cases/containers/unassigned-cases/unassigned-cases.component.spec.ts:60]: missing ;

File: src/unassigned-cases/containers/unassigned-cases/unassigned-cases.component.spec.ts [hunk 8]
```diff
@@ -1,16 +1,126 @@
...
+
+  beforeEach(() => {
+      mockStore = jasmine.createSpyObj('Store', ['pipe', 'dispatch']);
+      const actions$ = of (
+        [
+          UnassignedCasesActions.LoadUnassignedCaseTypes, UnassignedShareCasesActions.AddShareCases,
+          UnassignedShareCasesActions.SynchronizeStateToStore, UnassignedCasesActions.LoadUnassignedCases
+        ])
```
Comment (rayliangatos) [inline src/unassigned-cases/containers/unassigned-cases/unassigned-cases.component.spec.ts:68]: missing ;

File: src/users/containers/invite-user/invite-user.component.spec.ts [hunk 9]
```diff
@@ -1,14 +1,79 @@
...
+import * as fromAppStore from '../../../app/store';
+import * as fromStore from '../../store';
 
 describe('User Details Component', () => {
-
     let component: InviteUserComponent;
-    let userStoreSpyObject;
+    let mockStore: any;
+    let fixture: ComponentFixture<InviteUserComponent>;
+    let superNgOnDestroy: jasmine.Spy
```
Comment (rayliangatos) [inline src/users/containers/invite-user/invite-user.component.spec.ts:17]: missing ;

File: src/fee-accounts/containers/overview/account-overview.component.ts [hunk 10]
```diff
@@ -14,7 +14,7 @@ import { FeeAccount } from '../../models/pba-accounts';
 @Component({
   selector: 'app-prd-fee-accounts-component',
   templateUrl: './account-overview.component.html',
-})
+});
```
Comment (rayliangatos) [inline src/fee-accounts/containers/overview/account-overview.component.ts:17]: this ; needs to be removed, complication failed it breaks the start up of start:ng

**Review by johnbenjamin-hmcts — APPROVED** [review]

File: src/fee-accounts/containers/overview/account-overview.component.spec.ts [hunk 11]
```diff
@@ -1,19 +1,140 @@
...
+    }));
 
     it('should create', () => {
-        expect(component).toBeTruthy();
+      expect(component).toBeTruthy();
     });
 
+    it('organisation subscribes with data', (done) => {
+      component.org$.subscribe(org => {
+        console.log('org', org);
```
Comment (kuda1992) [inline src/fee-accounts/containers/overview/account-overview.component.spec.ts:106]: Left over console.log remove please

File: src/fee-accounts/containers/overview/account-overview.component.spec.ts [hunk 12]
```diff
@@ -1,19 +1,140 @@
...
+    it('loading subscribes with data', (done) => {
+      component.loading$.subscribe(loading => {
+          expect(loading).toBeFalsy();
+          done();
+      });
+    });
+
+    it('accounts subscribes with data', (done) => {
+      component.accounts$.subscribe(accounts => {
+        console.log('accounts', accounts);
```
Comment (kuda1992) [inline src/fee-accounts/containers/overview/account-overview.component.spec.ts:121]: Another one

**Review by kuda1992 — APPROVED** [review]
Remove the console.logs but all fine.

**Pull request closed without merging**